making submit2 the default (#456)
CHTC/chtc-website-source: uw-research-computing/use-submit-node.md

Computing Facilitator, please fill out [this form](form.html).**

Recently created CHTC accounts will use our submit server 

> *submit2.chtc.wisc.edu*. Accounts created previous to July 1, 2022 may be on 
> either *submit1.chtc.wisc.edu* or *submit2.chtc.wisc.edu*. See our 
> [connecting guide](connecting.html) for more specifics. 
> 

Reviewer: ```suggestion
*submit2.chtc.wisc.edu*. Accounts created before July 1, 2022 may be on 
either *submit1.chtc.wisc.edu* or *submit2.chtc.wisc.edu*. See our 
[connecting guide](connecting.html) for additional information. 

```
